server/public/styles.css

```diff
@@ -88,8 +88,11 @@ body {
   left: 15px;
 }
 
-.mapboxgl-ctrl-bottom-right,
-.mapboxgl-ctrl-bottom-left {
+.map-container > .mapboxgl-control-container > .mapboxgl-ctrl-bottom-right {
+  bottom: 65px;
+}
+
+.map-container > .mapboxgl-control-container > .mapboxgl-ctrl-bottom-left {
   bottom: 65px;
 }
```

Comment:
If you are overriding the default styles supplied by MapBox, you should put them in a separate `overrides.css`.